### PR TITLE
integration: update integration docker compose file.

### PR DESCRIPTION
**Pull request by 3AceShowHand**
### What problem does this PR solve? 

* set log-level to `info` for non `tikv` component
* alter `--sort-dir` to `--data-dir` for ticdc component.

### Check List 

Tests 

 - No code

Code changes

Side effects

Related changes

### Release note 

```release-note
`None`.
```

### CODE REVIEW

**Review by overvenus — APPROVED** [review]
LGTM

**Review by hi-rustin — APPROVED** [review]

**Review by asddongmen — APPROVED** [review]

File: docker-compose-avro.yml [hunk 1]
```diff
@@ -44,10 +44,10 @@ services:
       - --addr=0.0.0.0:8300
       - --pd=http://upstream-pd:2379
       - --log-file=/logs/capturer0.log
-      - --log-level=debug
+      - --log-level=info
```
Comment (amyangfei) [inline docker-compose-avro.yml:47]: should we keep debug log level for ticdc

Comment (3AceShowHand) [reply docker-compose-avro.yml:47]: yes, this is helpful.

**Pull request merged**